packages/server/src/api/rest/index.ts

```diff
@@ -52,6 +39,18 @@ export type Options = {
      * Defaults to 100. Set to Infinity to disable pagination.
      */
     pageSize?: number;
+
+    /**
+     * The divider used to separate compound ID fields in the URL.
+     * Defaults to '_'.
+     */
+    idDivider?: string;
+
+    /**
+     * The charset used for URL segment values. Defaults to `a-zA-Z0-9-_~ %`.
+     * If the idDivider is set to a different value, it should be included in the charset.
+     */
+    urlSegmentNameCharset?: string;
```

Comment:
Thinking about it again, I'll rename it to `urlSegmentCharSeet` for clarity. I'll make the change, merge the PR and make a new patch release. Thanks @thomassnielsen !